_events/2019/pearc-2019.md

```diff
@@ -0,0 +1,9 @@
+---
+title: PEARC19
+location: Chicago, IL
+url: https://www.pearc19.pearc.org/
+expires: 2019-08-01
```

Comment:
I think this is missing the date field "date: 2019-07-29"

Reply:
There isn't a date field - The date would be written in the content area by the author. The only reason we need to have an expires date is to know when the event should be automatically archived (removed from Events and Training, and added to Archive). In fact, if you manually set a date for the future, it will remove the post entirely (Jekyll by default only renders things that aren't written in the future.)

Reply:
The reason I assumed the date field was missing was that the archive page displayed today's date and not the event's archived date.  
Like this:
https://168-120906408-gh.circle-artifacts.com/0/usrse.github.io/events-archive/index.html

Looking at the code it looked like it would use event.date, and when I added the above date, the archived date showed up in the list.

Reply:
Ah good point, that's a bug! In fact it does use the event date, and only because I mirrored functionality of posts (where we would want the post date) and forgot we don't want that. Could you give me the start date for both of the events? I'll need to add a start_date (or similar) that can render.

Reply:
okay just pushed changes to fix these bugs, and add the date across the events/training page, archive, and single events page. Take a look!